dev-docs/development-process.md

```diff
@@ -153,6 +154,9 @@ Now follow along:
    be encrypted at rest.
    * SubscriptionId can be found in the Azure console
    * RabbitMQ account creds are in passwordstore at tc/cloudamqp.com/hip-macaw
+1. Run `yarn dev:db:upgrade` to upgrade the DB to the current version of the database.  You will generally want to do this
```

Comment:
Just noticed the following error:

```
➜  taskcluster git:(tc-lib-postgres) yarn dev:db:upgrade
yarn run v1.22.0
$ node infrastructure/tooling/src/main.js dev:db:upgrade
Error: invalid subcommand for db/src/main.js
    at main (/Users/hassanali/Documents/Mozilla/projects/taskcluster/db/src/main.js:29:11)
    at dbUpgrade (/Users/hassanali/Documents/Mozilla/projects/taskcluster/infrastructure/tooling/src/dev/index.js:63:43)
```

It's probably because `argv[2]` is `dev:db:upgrade` but the source code only checks for `upgrade`. Same issue with downgrade.

Reply:
Whoops, a conflict with the downgrade support!  I'll fix it up..

Reply:
41efd3ff5bc631e866c57c6e8b03312d89e33db6